circle-node-awscli/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM circleci/node:$NODE_VERSION
 RUN sudo apt-get update && \
   sudo apt-get install -y python python-pip python-dev && \
   sudo pip install awscli && \
+  sudo pip install awsebcli --upgrade --user && \
```

Comment:
Don't think the `user` flag is needed since we're installing within the container, not on the host.